benchmarks/continuous.py

```diff
@@ -89,15 +91,53 @@ def numpy_sum():
   return np.sum(xs * xs)
 
 
+def numpy_gaussian(n):
+  return lambda: np.random.normal(size=n)
+
+
+def numpy_matmul(n, width):
+  m1 = np.random.normal(size=(width, n, n)).astype(np.float32)
+  m2 = np.random.normal(size=(width, n, n)).astype(np.float32)
+  return lambda: np.matmul(m1, m2)
+
+
+def numpy_matvec(n, width):
+  ms = np.random.normal(size=(width, n, n)).astype(np.float32)
+  vs = np.random.normal(size=(width, n)).astype(np.float32)
+  return lambda: np.einsum('...ij,...j', ms, vs)
+
+
+def numpy_poly(n):
+  xs = np.arange(n, dtype=np.float32)
+  return lambda: 4.0 * (xs * xs * xs * xs) + 3.0 * (xs * xs * xs) + 2.0 * (xs * xs) + xs
```

Comment:
This might be slightly unfair to numpy since the lower powers are evaluated repeatedly. Perhaps use Horner's method?

Reply:
There's also [`numpy.polyval`](https://numpy.org/doc/stable/reference/generated/numpy.polyval.html) that might be a better baseline.

Reply:
Changed to use `np.polynomial.Polynomial`.